plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryArrowBufferAllocator.java

```diff
@@ -62,7 +62,7 @@ public boolean onFailedAllocation(long size, AllocationOutcome outcome)
             outcome.getDetails().ifPresent(details -> {
                 log.warn("Allocation failure details: %s", details.toString());
             });
-            return false;
+            return true;
```

Comment:
drop this change. It will not be effective anyway. It is used like this:
```
if (!outcome.isOk()) {
      if (listener.onFailedAllocation(actualRequestSize, outcome)) {
        // Second try, in case the listener can do something about it
        outcome = this.allocateBytes(actualRequestSize);
      }
      if (!outcome.isOk()) {
        throw new OutOfMemoryException(
            createErrorMsg(this, actualRequestSize, initialRequestSize), outcome.getDetails());
      }
    }
```

Given fact that we do not do anything to make another request successful it will fail anyway. 

Also rename the class

Reply:
ok!